src/DIRAC/Core/Utilities/DIRACScript.py

```diff
@@ -1,185 +1,2 @@
-            gMonitor.setComponentName(cls.scriptName)
-            gMonitor.setComponentLocation("script")
-            gMonitor.initialize()
-        else:
-            gMonitor.disable()
-        includeExtensionErrors()
-        return True
-
-    @classmethod
-    def showHelp(cls, dummy=False, exitCode=0):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.showHelp`."""
-        return cls.localCfg.showHelp(dummy=dummy, exitCode=exitCode)
-
-    @classmethod
-    def registerSwitches(cls, switches):
-        """Register switches
-
-        :param list switches: switches
-        """
-        for switch in switches:
-            cls.registerSwitch(*switch)
-
-    @classmethod
-    def registerSwitch(cls, showKey, longKey, helpString, callback=False):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.registerCmdOpt`."""
-        cls.localCfg.registerCmdOpt(showKey, longKey, helpString, callback)
-
-    @classmethod
-    def registerArgument(cls, description, mandatory=True, values=None, default=None):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.registerCmdArg`."""
-        cls.localCfg.registerCmdArg(description, mandatory, values, default)
-
-    @classmethod
-    def getPositionalArgs(cls, group=False):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.getPositionalArguments`."""
-        return cls.localCfg.getPositionalArguments(group)
-
-    @classmethod
-    def getExtraCLICFGFiles(cls):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.getExtraCLICFGFiles`."""
-        return cls.localCfg.getExtraCLICFGFiles()
-
-    @classmethod
-    def getUnprocessedSwitches(cls):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.getUnprocessedSwitches`."""
-        return cls.localCfg.getUnprocessedSwitches()
-
-    @classmethod
-    def addDefaultOptionValue(cls, option, value):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.addDefaultEntry`."""
-        cls.localCfg.addDefaultEntry(option, value)
-
-    @classmethod
-    def setUsageMessage(cls, usageMessage):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.setUsageMessage`."""
-        cls.localCfg.setUsageMessage(usageMessage)
-
-    @classmethod
-    def disableCS(cls):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.disableCS`."""
-        cls.localCfg.disableCS()
-
-    @classmethod
-    def enableCS(cls):
-        """See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.enableCS`."""
-        return cls.localCfg.enableCS()
+""" For backward compatability """
+from DIRAC.Core.Base.Script import Script as DIRACScript
```

Comment:
Importing this should show a deprecation message

Reply:
Ill create separate PR with it

Reply:
#5870